src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: $card-width;
+  background-color: $background-card-and-btn;
+  border-radius: 5px;
+  border: 1px solid $border-card;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  color: $main-color-text;
+  padding: $padding-card;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    padding-top: $padding-card;
+    align-self: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-top: 40px;
```

Comment:
why did you set this as a padding?
https://blog.hubspot.com/website/css-margin-vs-padding#:~:text=In%20CSS%2C%20a%20margin%20is,the%20space%20inside%20an%20element.